pyshtools/meson.build

```diff
@@ -0,0 +1,51 @@
+python_sources = [
+  '__init__.py',
+  '_version.py',
+]
+
+pymod = import('python')
+python = pymod.find_installation(pure: false, modules: ['numpy'])
+
+# NumPy include directory
+incdir_numpy = run_command(python,
+  [
+    '-c',
+    'import os; os.chdir(".."); import numpy; print(numpy.get_include())'
```

Comment:
This has issues with in-tree virtualenvs, which is an annoying problem. You may want to take over https://github.com/scipy/scipy/blob/1a3021a08e5898b52d7b98f7b1b21ae4a6306484/scipy/meson.build#L56-L82 here.

Reply:
I guess this requires running with `--no-build-isolation`, because I don't have this problem by default.

Reply:
Yes, I only have in-tree venvs.